app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="min-h-screen bg-gray-50 flex flex-col justify-center py-12 sm:px-6 lg:px-8">
+  <div class="sm:mx-auto sm:w-full sm:max-w-md">
+    <h2 class="mt-6 text-center text-3xl leading-9 font-extrabold text-gray-900">
+      Sign in to your account
+    </h2>
+
+  </div>
+
+  <div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
+    <div class="bg-white py-8 px-4 shadow sm:rounded-lg sm:px-10">
+      <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+      <div>
+        <%= f.label :email, class: "block text-sm font-medium leading-5 text-gray-700" %>
```

Comment:
the rails forms helpers are unstyled so you can just drop classes on them like this. After 15 minutes looking up the syntax again, it was pretty simple :)